behat/tests/features-v2/cross-browser/js-basic-test.feature

```diff
@@ -0,0 +1,8 @@
+@v2 @js-basic-check
```

Comment:
Could we get an extra `@js` tag here too? There are some built in behat features to do with selecting drivers we may want to hook in to in the future that relies on a plain ole `@js` tag to work.

Reply:
Done